### PR TITLE
#38 Set package metadata

### LINKED ISSUE #38: Switch to license expression & set repo url
### Summary
I wish for the nuget packages to have the licence expression property set correctly and the repository url

### Details

The licence expression property should be set to the correct licence type I.e. Apache-2.0 as this will enable analysis of licences in use to occur in external tools & the license type will be shown in Nuget etc. By providing the repo url it will make it easier to contribute and can be used by source-link.

### PR DESCRIPTION
**Pull request by thompson-tomo**
Closes #38

### CODE REVIEW

File: StatsigRedis/StatsigRedis.csproj [hunk 1]
```diff
@@ -9,8 +9,9 @@
         <Version>1.0.0</Version>
         <Authors>Statsig Inc.</Authors>
         <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+        <RepositoryUrl>https://github.com/statsig-io/dotnet-sdk.git</RepositoryUrl>
         <Owners>Statsig Inc.,</Owners>
-        <PackageLicenseFile>LICENSE</PackageLicenseFile>
+        <PackageLicenseExpression>MIT</PackageLicenseExpression>
```
Comment (tore-statsig) [inline StatsigRedis/StatsigRedis.csproj]: Adding the license expression is great, but this repo and all of our sdks are under the ISC license

https://github.com/statsig-io/dotnet-sdk/blob/main/LICENSE

File: dotnet-statsig/dotnet-statsig.csproj [hunk 2]
```diff
@@ -10,8 +10,9 @@
     <Version>1.25.0</Version>
     <Authors>Statsig Inc.</Authors>
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+    <RepositoryUrl>https://github.com/statsig-io/dotnet-sdk.git</RepositoryUrl>
     <Owners>Statsig Inc.,</Owners>
-    <PackageLicenseFile>LICENSE</PackageLicenseFile>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
```
Comment (tore-statsig) [inline dotnet-statsig/dotnet-statsig.csproj]: ISC

Comment (thompson-tomo) [reply dotnet-statsig/dotnet-statsig.csproj]: All fixed

File: StatsigRedis/StatsigRedis.csproj (hunk 1 above)
Comment (thompson-tomo) [reply StatsigRedis/StatsigRedis.csproj]: All fixed

**Pull request closed without merging**